pkg/apis/serving/v1beta1/inference_service_validation_test.go

```diff
@@ -26,6 +26,32 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+func makeTestRawInferenceService() InferenceService {
+	inferenceservice := InferenceService{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "foo",
+			Namespace: "default",
+			Annotations: map[string]string{
+				"serving.kserve.io/deploymentMode":              "RawDeployment",
+				"serving.kserve.io/autoscalerClass":             "hpa",
+				"serving.kserve.io/metrics":                     "cpu",
+				"serving.kserve.io/targetUtilizationPercentage": "75",
```

Comment:
Can we also validate the `targetUtilizationPercentage` ? it should be in (0,100).